netci.groovy

```diff
@@ -374,7 +430,7 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                         }
                     }
                     // For x86, only add per-commit jobs for Windows
-                    else if (architecture == 'x86ryujit' || architecture == 'x86lb') {
+                    else if (architecture == 'x86' || architecture == 'x86compatjit' || architecture == 'x86lb') {
```

Comment:
I agree. We should back this off if these are for triggers. In this case, I just replaced the 2 existing with the 3 new archs. I'll call this a to-do.